index.js

```diff
@@ -19,6 +22,8 @@ app.use("/role", roleRouter);
 
 app.use("/user", userRouter);
 
+app.use("/token", tokenRouter);
```

Comment:
this route is not needed, as per the assignment you need to create this token at the time of login. so this will be a helper/utility function

Reply:
then where i apply token function

Reply:
Read the comment properly

Reply:
apologies sir but i am still not getting how to do it.